parley/src/layout/cluster.rs

```diff
@@ -21,7 +21,10 @@ impl<'a, B: Brush> Cluster<'a, B> {
         let mut path = ClusterPath::default();
         if let Some((line_index, line)) = layout.line_for_byte_index(byte_index) {
             path.line_index = line_index as u32;
-            for (run_index, run) in line.runs().enumerate() {
+            for run_index in 0..line.len() {
```

Comment:
Unfortunately we can't just use `line.items()`, as `GlyphRunIter` currently appears to skip the run on empty lines.